### DISCUSSION TITLE
Will laravel 11 Scheduler be Macroable?

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#50348] [Q&A] [answered]
Hi, according to https://laravel.com/docs/master/releases#scheduling,  the Facade Illuminate\Support\Facades\Schedule can be used.
When using backgrournd scheduled commands and stdout logging, in order to make the logs work I had to create a macro customCommand (https://github.com/laravel/framework/discussions/50085#discussioncomment-8495648). Will this be possible in laravel 11?

**Comment by henzeb** [comment, marked as answer]
Looking at the code the Schedule will still be Macroable
